Services/Data/MyDBContext.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Microsoft.Data.SqlClient;
```

Comment:
1. Naming Conventions
Ensure consistent naming conventions for methods and properties. Method names should be in PascalCase.
Consider renaming sp_AddProduct, sp_GetProducts, and sp_ExportExcelCategory to AddProduct, GetProducts, and ExportExcelByCategory for consistency with C# naming conventions.

Reply:
Resolved